tracing-rfc-5424/src/byte-utils.rs

```diff
@@ -22,5 +22,5 @@ pub fn bytes_from_os_str(s: std::ffi::OsString) -> Vec<u8> {
 
 #[cfg(not(unix))]
 pub fn bytes_from_os_str(s: std::ffi::OsString) -> Vec<u8> {
-    unimplemented!("bytes_from_os_str is not supported on non-Unix.");
+    s.to_string_lossy().as_bytes().to_vec()
```

Comment:
The problem here is that, AFAIU, on Windows, `s` may contain a UTF-16 string. In that case, `to_string_lossy()` will likely mangle the text quite badly.

Reply:
OsStr::to_string_lossy() is a safe function, and this crate works well in windows platform.
If there is any other non-compatibility, I will report to you. Thanks for your library.